### PR TITLE
Database schema

### LINKED ISSUE #3: Create Database Schemas
**As a** Promotions Manager,
**I need** to have a database to persist data of product promotion policies,
**So that** I do not lose the input promotion policies after the server is rebooted.
   
 ### Details and Assumptions
 * We will use PostgreSQL as the persistent store.
 * There should be **one** schema (tables):
    - The first one should have fields including promotionID, promotionName, promotionCode, promotionStartDate, promotionEndDate, promotionType, and description. Here, promotionID is an auto-incrementing (primary) key maintained by the database itself rather than being input by the manager. 
 * The schema should have a foreign key product_id.
 ### Acceptance Criteria  
   
 ```gherkin
 Given I implemented all the required database schemas,
 When a promotion policy is created, updated, or deleted for a product,
 And I reboot the service,
 And I search for the promotion policy for the product,
 Then I should see the change to that product still remains.
 ```

### PR DESCRIPTION
**Pull request by tomasort**
Added models for Products and Promotions

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -164,28 +186,12 @@ def find_by_name(cls, name):
...
------------
-id (int) - the id of the prodcut 
-name (string) - the name of the product 
-price () - the price of the product 
-units (int) - the number of units available of a specific product 
-
-"""
-
 
 class Product(db.Model):
```
Comment (yaozcoderepo) [inline service/models.py]: I think we do not need to maintain a table of `Product`. Instead, we just need a `product_id`.

**Comment by tomasort** [comment]
Removed everything related to the product table. Now there is only one product_id per row.

**Comment by yaozcoderepo** [comment]
We may also need to fill out two methods and add one more method.
two methods: serialize and deserialize methods (we need to use these two methods when implementing CRUD services)
one method: find_by_product_id method. (when implementing the "list promotion" service, we need to find the corresponding promotion by the product_id).

You can choose to update them in this pull request. Or, you can just put some comments (#todo) here so that you can implement them in your next pull request.

**Comment by tomasort** [comment]
> We may also need to fill out two methods and add one more method. two methods: serialize and deserialize methods (we need to use these two methods when implementing CRUD services) one method: find_by_product_id method. (when implementing the "list promotion" service, we need to find the corresponding promotion by the product_id).
> 
> You can choose to update them in this pull request. Or, you can just put some comments (#todo) here so that you can implement them in your next pull request.

I don't think these issues are part of this user story.

**Comment by yaozcoderepo** [comment]
It's okay. I can start a new pull request for these three functions.

**Pull request merged**